run.sh

```diff
@@ -0,0 +1,5 @@
+clear
```

Comment:
We already have a make file, if some sort of automation is needed. Also this file hardcodes personal file locations. 

Pls always check before sharing link.

Reply:
Yes forgot to check, will do.

Reply:
Delete the file from the MR?

Reply:
Forgot :|